content/en/APIUseCases/_index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "API Use Cases"
+linkTitle: "API Use Cases"
```

Comment:
API Guides

I think "use cases" is too technical.

Reply:
```suggestion
title: "API Guides"
linkTitle: "API Guides"
```

Reply:
I've browsed some "best in class" API docs, and I see both "use cases" and "guides".

I appreciate your perspective, but IMO "Use Cases" is better, as it promises specific solutions. In contrast, "Guides" feels like a book that someone has to study.

Reply:
No problem, both are good.